pkg/monitortests/node/watchnodes/monitortest.go

```diff
@@ -42,6 +42,19 @@ func (*nodeWatcher) ConstructComputedIntervals(ctx context.Context, startingInte
 }
 
 func (*nodeWatcher) EvaluateTestsFromConstructedIntervals(ctx context.Context, finalIntervals monitorapi.Intervals) ([]*junitapi.JUnitTestCase, error) {
+	// Fail tests when monitor test flags this as an error
+	for _, val := range finalIntervals {
+		if val.Source == monitorapi.SourceNodeMonitor && val.Message.Reason == monitorapi.NodeUnexpectedReadyReason {
+			return []*junitapi.JUnitTestCase{{
+				Name: "UnexpectedNodeNotReady", SystemOut: val.Message.HumanMessage,
+			}}, nil
+		}
+		if val.Source == monitorapi.SourceNodeMonitor && val.Message.Reason == monitorapi.NodeUnreachableReason {
+			return []*junitapi.JUnitTestCase{{
+				Name: "Unreachable", SystemOut: val.Message.HumanMessage,
+			}}, nil
+		}
+	}
```

Comment:
a few things here.

1. to fail a test, the failureoutput in the junit test case must be set
2. a test should only fail once per run
3. a test must either fail or succeed, it cannot be missing entirely.  This means the test needs to be added in success cases.
4. tests should follow a format like `[Jira:"Node / Kubelet"] nodes should not go unready unexpectedly`  The Jira bit automatically categorizes ownership and the test is descriptive

I noticed when checking https://prow.ci.openshift.org/view/gs/test-platform-results/pr-logs/pull/28989/pull-ci-openshift-origin-master-e2e-metal-ipi-ovn-kube-apiserver-rollout/1823032010419998720

Reply:
Ah thanks for catching that. I will add this.

Reply:
Pushed a fix.